drivers/rtaudio/audio_driver_rtaudio.cpp

```diff
@@ -32,8 +32,15 @@ const char* AudioDriverRtAudio::get_name() const {
 int AudioDriverRtAudio::callback( void *outputBuffer, void *inputBuffer, unsigned int nBufferFrames,
 	double streamTime, RtAudioStreamStatus status, void *userData ) {
 
-	if (status)
-		print_line("lost?");
+	if (status) {
+		if (status == RTAUDIO_INPUT_OVERFLOW) {
+			WARN_PRINT("RtAudio input overflow!");
+		}else if (status == RTAUDIO_INPUT_UNDERFLOW) {
```

Comment:
That wouldn't work as the constant is `RTAUDIO_OUTPUT_UNDERFLOW`.
Looking at the RtAudio API the fallback is probably not needed, the status can be either 0, `RTAUDIO_INPUT_OVERFLOW`, `RTAUDIO_OUTPUT_UNDERFLOW`, or both (it's a flag). But there doesn't seem to be a case where both flags are set at the same time as input and output seem mutually exclusive.